images/macos/provision/core/Configure-Toolset.ps1

```diff
@@ -0,0 +1,60 @@
+################################################################################
+##  File:  Configure-Toolset.ps1
```

Comment:
All files in the `images/macos/provision/core` folder are named in lowercase. Please change the name to `configure-toolset.ps1`

Reply:
Renamed it